src/pages/Reader.tsx

```diff
@@ -130,17 +134,27 @@ class Reader extends React.Component<any, State> {
     this.setState(curState => ({
       ...curState,
       page: newPage,
-    }), () => !scroll && this.myRef?.current?.scrollIntoView())
+    }), () => (!scroll && this.state.loaded) && this.myRef?.current?.scrollIntoView())
     
   }
 
   checkVisible = (inView: boolean, ind: number) => {
     if (inView) {
+      console.log('dafkdfjls')
       this.updatePage(ind, true)
     }
   }
 
 
+  toClasses = (options: string[]) => {

```

Comment:
i get why you need this but its sorta yucky.

having the enum value of readerMode, imageSizing be the display text is a bit weird.
alternatively, just make the enums numbers:
```
export enum ReaderMode {
  longStrip=1,
  scrolling,
  ...
}
```
and then have a fixed map:
```
ReaderMode.longStrip: {
   displayText: "Long Strip"
   class: "long-strip"
}
```
and then when you actually use the css class:
```
className="${imageSizingMap[fitWidth].class} ${readerModeMap[longStrip].class}"
```

this might be overkill tho

Reply:
yeah i get what u mean. might implement if the display stuff starts gettng wonky

Reply:
and as an extra bonus, you could also abstract out all the ui text into one big map:
```
{
  'SETTINGS-TEXT': 'settings',
  'SHARE-LABEL': 'share'
}
```
this would be nice if ever wanna add multilnaguage support

Reply:
Is there some i18n package that simplifies things

Reply:
the i18n package basically does it that way